bokehjs/src/coffee/models/tools/inspectors/hover_tool.coffee

```diff
@@ -147,9 +147,8 @@ export class HoverToolView extends InspectToolView
 
     for i in indices['1d'].indices
       # multiglyphs will set '1d' and '2d' results, but have different tooltips
-      if not _.isEmpty(indices['2d'])
-        for pair in _.pairs(indices['2d'])
-          [i, j] = [pair[0], pair[1][0]]
+      if not isEmpty(indices['2d'])
+        for i, [j] of indices['2d']
```

Comment:
is this completely equivalent? What will this do if the second array is longer than length 1?